hadoop-tools/hadoop-aws/src/test/resources/core-site.xml

```diff
@@ -153,11 +153,11 @@
   <!-- Reduce DDB capacity on auto-created tables, to keep bills down. -->
   <property>
     <name>fs.s3a.s3guard.ddb.table.capacity.read</name>
-    <value>10</value>
```

Comment:
wow, I'd missed completely we were setting values here

These need to be changed to 0, to ensure the tables are always on demand